Gaijinizer patch/Map003.txt

```diff
@@ -1460,29 +1459,30 @@ as usual.#MTLed
 
 ３ターンのうちに、１０００以上削れば全消しだ。
 > CONTEXT: Map003/events/1/pages/2/371/Dialogue/クッション君
-The recovery amount of the other's self
-regeneration is also. \\C[2] Hyakki neutralizes the
-item every turn. If you cut 1000 or more in 3
-turns, it's all gone.#MTLed
+Hyakki will also \\C[2]counteract your
+opponent's regeneration with her items every turn.
+If you can shave off 1000 or more HP within 3
+turns, it will be completely neutralized.#Strate
 > END STRING
```

Comment:
Note

Reply:
You mean in "it will be completely neutralized"? Pretty sure the "it" is referring to the regeneration. Hyakki deals about 3000 damage per turn, Titanoboa heals 9999 damage every 3 turns, so Hyakki's 9000 total damage + at least 1000 from you completely offsets the heal. I think that's what this text is trying to explain.

Reply:
Ah, my bad. Looks good as is, then 👍